cmd/policy/main.go

```diff
@@ -116,7 +116,7 @@ func (g *Globals) setup() func() {
 	configFile := g.Config
 
 	policyAPP, cleanup, err := app.BuildPolicyApp(
-		os.Stdout,
+		os.Stderr,
```

Comment:
This doesn't look ok - I think you're printing everything on stderr now?

Reply:
Yes, but just everything that zerolog logs